src/test/java/org/zalando/riptide/AsyncTest.java

```diff
@@ -75,6 +75,20 @@ public void shouldCall() {
     }
 
     @Test
+    public void shouldCallWithoutParameters() {
```

Comment:
Though I guess it doesn't really fit at this place ... I've misread the other `shouldCall...` test methods there.

I still don't find any place the previous UntypedCondition.call is explicitly tested (it is used in other tests to see which of the conditions matched).